reanalysis/query_panelapp.py

```diff
@@ -177,9 +177,28 @@ def gene_list_differences(latest_content: PanelData, previous_genes: Set[str]):
             latest_content[gene_ensg]['new'] = True
 
 
+def write_output_json(output_path: str, object_to_write: Any):
+    """
+    writes object to a json file
+    AnyPath provides platform abstraction
+
+    :param output_path:
+    :param object_to_write:=
+    """
+
+    logging.info('Writing output JSON file to %s', output_path)
```

Comment:
```suggestion
    logging.info(f'Writing output JSON file to {output_path}')
```

Also in a few other places, unless you're keen on the call to optimize out the interpolation in case the logging level is filtered.

Reply:
I was on the fence about this... I haven't (yet) posted in a mix of info and debug statements, but one useful debug one would be periodic calls to `matrix.row_count()` to assess the relative impact of each filtering step, which can involve massive computation just for a row count so would be debug only.

The main reason for using this syntax is that pylint was complaining...

Reply:
Yeah, I think that would even be interesting at the `info` level. `row_count` shouldn't be expensive.

Reply:
It was definitely time-expensive in the notebook I was running, which is probably the wrong point of reference 😆